resources/views/tags/entities/_form.blade.php

```diff
@@ -1,14 +1,17 @@
 <?php /** @var \App\Models\Tag $model */?>
 {{ csrf_field() }}
 <x-grid type="1/1">
-    <x-forms.field field="entities" :required="true" >
-        <x-form.entities 
-            :options="['exclude-entity' => $model->entity->id, 'route' => 'search.tag-children']" 
-            :campaign="$campaign"
-            :ajax="request()->ajax()"
-        >
-        </x-form.entities>
-    </x-forms.field>
+    <x-forms.foreign
+        field="entities"
+        :required="true"
+        label="abilities.show.tabs.entities"
+        multiple="multiple"
```

Comment:
I feel like multiple should be a boolean, and validated on the Foreign.php component